dashboard/config/scripts_json/csd3-2023.script_json

```diff
@@ -15,7 +16,7 @@
       "weekly_instructional_minutes": 225
     },
     "new_name": null,
-    "family_name": "csd3",
+    "family_name": null,
     "serialized_at": "2023-03-17 23:30:18 UTC",
     "published_state": "in_development",
```

Comment:
FYI, whenever you add these to a course, you'll need to remove the `published_state` from each of these units. It won't cause any errors, so it can be done as a follow-up, but it'll cause confusion when we go to publish the course. We do allow units within a course to have a different `published_state` from the course in order to allow curriculum writers to publish units as they go (this was needed for the CSA pilot), but that's not what CSD needs.